### ISSUE TITLE
Chinese fonts render characters without proper spacing

### ISSUE THREAD

**Issue by dhowe** [nodebox/opentype.js#280]
This seems to be specific to .otf fonts (and happens with english or chinese text):

![image](https://cloud.githubusercontent.com/assets/737638/25130490/e945c7e6-2474-11e7-889a-5875f342c76e.png)

[Fangsong.otf.zip](https://github.com/nodebox/opentype.js/files/929341/Fangsong.otf.zip)

Code here:

```
<html>
<head>
  <meta charset="UTF-8">
  <script src="lib/opentype.js"></script>
  <script>
  window.onload = function() {
    opentype.load('fonts/FangTong.otf', function(err, font) {
      if (err) {
        alert('Font could not be loaded: ' + err);
      } else {
        var ctx = document.getElementById('canvas').getContext('2d');
        var path = font.getPath('Hello, World!', 0, 100, 22);
        path.draw(ctx);
      }
    });
  }
  </script>
</head>
<body>
<canvas id="canvas" style="width: 600px"></canvas>
</body>
</html>
```

**Comment by Jolg42** [comment]
It looks like it's related to that issue https://github.com/nodebox/opentype.js/issues/273

**Comment by fdb** [comment]
This is fixed as of version 0.7.0.
